frontend/public/components/project-overview.jsx

```diff
@@ -5,8 +5,18 @@ import * as classnames from 'classnames';
 import { Link } from 'react-router-dom';
 import { ListView } from 'patternfly-react';
 
-import { ResourceLink, resourcePath } from './utils';
+import { ResourceIcon, resourceObjPath, resourcePath } from './utils';
+
+const getRevision = (obj) => {
+  const annotation = obj.kind === 'ReplicaSet'
+    ? 'deployment.kubernetes.io/revision'
+    : 'openshift.io/deployment-config.latest-version';
+  const revision = _.get(obj, ['metadata', 'annotations', annotation]);
+  return revision ? `#${revision}` : obj.metadata.name;
+};
 
+const ControllerLink = ({controller}) =>
+  <Link to={resourceObjPath(controller, controller.kind)} title={controller.metadata.name}>{getRevision(controller)}</Link>;
```

Comment:
Since we can't reliably determine what the actual active controller is, should we just link to the appropriate list page?

Reply:
I think if we're going to show a specific number here, we should link to that object.

We might revisit how we're determining what revision is active, however. (For deployment configs at least, I believe there is a way to determine what is the current active replication controller.)